src/book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author

```

Comment:
- [ ] Kindly add the getter-setter (`attr_accessor`) for the `:rentals` variable.

Reply:
resolved 👍